src/main/target/CRAZYBEEF3FR/target.h

```diff
@@ -26,6 +26,9 @@
 #elif defined(CRAZYBEEF3DX)
 #define TARGET_BOARD_IDENTIFIER "CBDX"
 #define USBD_PRODUCT_STRING     "CrazyBee F3 DX"
+#elif defined(CRAZYBEEF3DXSPI)
+#define TARGET_BOARD_IDENTIFIER "CBDS"
```

Comment:
Is this actually a board that is being manufactured / sold?

Reply:
No, I believe I own the only two bards in existence as of now.

Reply:
Probably a good idea to add this to a different target instead:
- there is already a CRAZYBEEF3 target with a Spektrum RX - having 2 of them to choose from will be confusing for users;
- adding this to a target that has SPI pins exposed on a header (OMNIBUSF4FW, BEEROTORF4, MATEKFxx?, ...) will make it easier for other users to add their homebrew SPI RX mods.

You might want to talk to @githubDLG though, he might be interested in producing a board with a Spektrum SPI RX.

Reply:
hehe,  hi, mikeller and phobos~~~
Actually,  we and phobos have already work together  on this spi dsmx for a while.
And, the previous crazybee f3 dx is a intergrated serial rx, while this new `crazybee f3 dx spi` will acess the rfic directly via spi. 
Including the two variant in one  target maybe a natually thing, but seperate `crazybee f3 dx spi`  to another target will also acceptable then.
And, anyway , thanks for the work that your guys did!!! @mikeller  @phobos-

Reply:
@githubDLG: Good to hear that you are working together. I think we will have to be careful with how we are naming the target with Spektrum SPI RX - it's probably not possible to have a target that will automatically detect if the board uses SPI or an UART, so we will end up with two targets, and it will be important to make sure users know which target to use.

Reply:
I'll open a spearate PR with target specific changes, and we can discuss it there. Since the board is not produced target definition is very flexible. It is up to you @mikeller and @githubDLG , but please agree on some standard so that I can work on it and prepare the code.

@mikeller  right now it is a separate variant CRAZYBEEF3DX**SPI** (versus CRAZYBEEF3DX for serial rx). Maybe the SPI target should use the same pattern (two letters after F3) but with different characters?